### ISSUE TITLE
NavCard arrow pushed to the far right with large text size

### ISSUE THREAD

**Issue by knarangerel** [Fire-app/fire-mobile#41]
With large text size on Android the NavCard arrow gets pushed to the far right

![Screenshot_20200727-230637](https://user-images.githubusercontent.com/31663746/88615897-c9d77580-d060-11ea-954e-8d906742a367.jpg)

**Comment by kiwansim** [comment]
Same issue can be replicated on iOS as well:
https://tuftscscodeforgood.slack.com/files/U014XMY3V5F/F0181PX1HFW/image_from_ios.png?origin_team=T012S4H091Q&origin_channel=D015KJRMEQG
